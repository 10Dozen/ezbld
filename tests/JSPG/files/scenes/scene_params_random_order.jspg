# SceneParamsRandomOrder

Text line in the beginning.

*goto: "Scene2"

Text line between params.

*post_exec: Foo.Bar()
**pre_exec:
    let foo = bar()
    foo.update(10)
**

Final text line here.
