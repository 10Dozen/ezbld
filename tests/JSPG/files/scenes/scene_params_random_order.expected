[0]
cls = JSPGScene
name = SceneParamsRandomOrder
goto = "Scene2"
post_exec = Foo.Bar()
pre_exec =
    \tlet foo = bar()
    \tfoo.update(10)

desc =
    Text line in the beginning.

    Text line between params.

    Final text line here.



